src/states/elder/mod.rs

```diff
@@ -2158,9 +2147,15 @@ impl Approved for Elder {
                 trusted = Some(ps);
             }
         }
-        if validates(&trusted, &sec_info) || self.is_trusted(&sec_info)? {
-            let _ = self.add_new_section(&sec_info);
+        if !(validates(&trusted, &sec_info) || self.is_trusted(&sec_info)?) {
+            trace!(
+                "{} Adding untrusted neighbour section info {:?}",
+                self,
+                &sec_info
+            );
         }
+        // Always add section info until SMD is complete
```

Comment:
shall we have a TODO keyword here to highlight?

Reply:
I prefer to be cautious with TODO as they quickly become outdated, essentially the TODO would be implement SMD.
We will definitely need to update this code for SMD, otherwise test that validate that we reject invalid update should fail.

Reply:
I suggest to use a keyword of `TODO` just for a highlight so we won't forget to update this place of code (and the comment as well).
If you think it won't be forgot for sure, I am ok with leave it.  :)